src/views/certifications/components/standalone-link-contents/standalone-link-contents.module.css

```diff
@@ -0,0 +1,25 @@
+.root {
+	align-items: center;
+	color: var(--token-color-foreground-strong);
+	display: flex;
+	font-weight: var(--token-typography-font-weight-medium);
+	gap: 6px;
+	line-height: var(--line-height);
+	margin: 0;
+
+	& svg {
+		flex-shrink: 0;
+	}
+}
+
+.small {
+	composes: hds-typography-body-100 from global;
+
+	--line-height: 1.231;
```

Comment:
Very specific line height 😆 I could've sworn at one point we had a standalone link type style somewhere? Was that the old system?

Reply:
I think that might've been the old system, but I'll try to double-check! I'd pulled this from `components/standalone-link`.

Reply:
Turns out I was wrong or imagining something earlier - `standalone-link` actually uses `1rem` for the line height, so I've updated this to match 👍 I couldn't find any standalone link type styles in HDS... What jumps to mind for me is [`.g-type-buttons-and-standalone-links`](https://github.com/hashicorp/mktg-global-styles/blob/main/components/g-type.css#L217) from the marketing design system.